### PR TITLE
Feature cases section

### PR DESCRIPTION
**Pull request by MaxAlter**
Add Details Platform page

### CODE REVIEW

File: pages/platform/id.module.css [hunk 1]
```diff
@@ -1,9 +1,117 @@
...
+  margin: 25px 0;
+}
+
+.imageContainer {
+  position: absolute;
+  right: 10px;
+  top: 23px;
+}
+
+.imageDiv {
```
Comment (zAndrey95) [inline pages/platform/id.module.css:22]: pages/platform/contain.module.css
Copy

File: components/cases/cases.module.css [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+  transition: background-color 0.3s ease-in-out;
+}
+
+.imageContainer {
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+}
+
+.imageDiv {
```
Comment (zAndrey95) [inline components/cases/cases.module.css:67]: pages/platform/contain.module.css
Copy

File: components/cases/cases.module.css [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+
+.desc {
+  color: #b3b3b3;
+  margin: 0 0 30px 0;
+  font-family: "Roboto Mono", monospace;
+  font-size: 16px;
+  line-height: 1.6;
+}
+
+.headText {
```
Comment (zAndrey95) [inline components/cases/cases.module.css:21]: pages/platform/id.module.css

Copy

**Review by zAndrey95 — COMMENTED** [review]
More copy style!

**Review by zAndrey95 — COMMENTED** [review]
More copy style!

File: pages/platform/id.module.css (hunk 1 above)
Comment (MaxAlter) [reply pages/platform/id.module.css:22]: done

File: components/cases/cases.module.css (hunk 2 above)
Comment (MaxAlter) [reply components/cases/cases.module.css:67]: done

File: components/cases/cases.module.css (hunk 3 above)
Comment (MaxAlter) [reply components/cases/cases.module.css:21]: done

**Review by zAndrey95 — APPROVED** [review]

**Pull request merged**